Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java

```diff
@@ -428,9 +427,6 @@ public void showSaveImageInformationDialogWhenStandalone(int permissionCode, int
 		}
 
 		boolean isStandard = false;
-		if (permissionCode == MainActivityConstants.PERMISSION_EXTERNAL_STORAGE_SAVE_COPY) {
```

Comment:
Can you put this part back in? With this the last failing test works as well, i just tested it locally.

Reply:
done